lib/cli.js

```diff
@@ -11,9 +11,25 @@ module.exports = async process => {
   // so now both broken and unsupported use console, but only broken
   // will process.exit. It is important to now perform *both* of these
   // checks as early as possible so the user gets the error message.
-  const { checkForBrokenNode, checkForUnsupportedNode } = require('./utils/unsupported.js')
-  checkForBrokenNode()
-  checkForUnsupportedNode()
+  const semver = require('semver')
+  const supported = require('../package.json').engines.node
+  const knownBroken = '<6.2.0 || 9 <9.3.0'
```

Comment:
we should change this to `< 12.5.0` as we use syntax that won't parse in versions less than that.

Reply:
Very good point. Updated.